core/domain/prod_validation_jobs_one_off.py

```diff
@@ -859,6 +873,20 @@ class UserStatsModelAuditOneOffJob(ProdValidationAuditOneOffJob):
     is a very large list.
     """
 
+    @classmethod
+    def enqueue(cls, job_id, additional_job_params=None):
+        """Marks a job as queued and adds it to a queue for processing.
+
+        Args:
+            job_id: str. The ID of the job to enqueue.
+            additional_job_params: dict(str : *) or None. Additional parameters
+                for the job.
+        """
+        # We have to decrease the number of shards for this job, since the job
+        # is getting killed due to Out Of Memory.
```

Comment:
Ditto.

Reply:
Since the error is occurring in the shuffle sort, it means that a lot of errors are occurring and it is not able compile all the errors. So once the errors are fixed, this problem should not occur. We will be fixing these errors in another PR. Thanks.

Reply:
Probably you should handle the errors existing too, though, since this is an audit job and is supposed to produce some sort of result regardless of what is happening.

One thing I suggested before (to @vojtechjelinek? I can't recall) was to find away to only pass along at most 100 errors somehow so as not to overload the MR. Is that possible?

Reply:
I had a discussion with @vojtechjelinek , we should be able to find a method to pass only the top 100 errors. But since this PR is almost done except this job. We think that we can make the changes in the next PR. Thanks.

Reply:
I have created the [issue](https://github.com/oppia/oppia/issues/11742). Thanks.